src/ui/PromotedResultsBadge/PromotedResultsBadge.ts

```diff
@@ -21,8 +21,8 @@ export interface IPromotedResultsBadgeOptions {
  * The `PromotedResultsBadge` component adds a badge to promoted results in your interface.
  *
  * To be considered promoted, a result needs to either:
- * - be a Featured Result configured through a Coveo Query Pipeline (see [Managing Query Pipeline Featured Results](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=126))
- * - be a recommended result by Coveo Machine Learning (see [Coveo Machine Learning Features](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=183)).
+ * - be a Featured Result configured through a Coveo Query Pipeline (see [Managing Query Pipeline Featured Results](https://docs.coveo.com/en/1961/))
+ * - be a recommended result by Coveo Machine Learning (see [Coveo Machine Learning Features](https://docs.coveo.com/en/1671/)).
```

Comment:
We should probably link to the #automatic-relevance-tuning-art-feature anchor in the page

Then the link name would be Automatic Relevance Tuning (ART) Feature